fvm/src/executor/mod.rs

```diff
@@ -70,6 +71,14 @@ pub struct ApplyRet {
     pub penalty: BigInt,
     /// Tip given to miner from message.
     pub miner_tip: BigInt,
+
+    // Gas stuffs
+    pub base_fee_burn: TokenAmount,
+    pub over_estimation_burn: TokenAmount,
+    pub refund: TokenAmount,
```

Comment:
Let's continue using BigInt here. TokenAmount is aliased to BigInt, so not a big deal, but I'd prefer to contain entropy.

Reply:
Hm. We should probably go the other way so we can eventually get rid of that alias, right?